src/main/notifications/index.js

```diff
@@ -72,3 +79,15 @@ export function displayDownloadCompleted(fileName, path, serverInfo) {
     });
     download.show();
 }
+
+function getDoNotDisturb() {
+    if (process.platform === 'win32') {
+        return getFocusAssist().value;
+    }
+
+    if (process.platform === 'darwin') {
+        return getDarwinDoNotDisturb();
+    }
+
+    return false;
```

Comment:
we might be able to take a look at gsettings for getting the state in linux (see https://github.com/vector-im/element-web/issues/7758#issuecomment-671433520, it mentions dconf but I think gnome3 migrated to gsettings)

It wont be perfect but we can probably check every now and then for the settings and update accordingly so most linux users won't experience this problem. it might be out of scope of this PR (which is already a great improvement), if so, we can fill a second ticket for it.

Reply:
I think we should make it a separate ticket, purely because I don't think this is standard and it might be a little finicky to test and get right. I can make a ticket for it.